src/Stack/Ide.hs

```diff
@@ -43,7 +43,7 @@ ide
     -> [String] -- ^ GHC options.
     -> m ()
 ide targets useropts = do
-    (_realTargets,_,pkgs) <- ghciSetup Nothing targets
+    (_realTargets,_,pkgs) <- ghciSetup (Just BSAll) Nothing targets
```

Comment:
@mgsloan: Should it stay with the old behaviour of not starting if the dependencies fail to build, or would the new behaviour of continuing anyway be better?